test/Test/Data.hs

```diff
@@ -18,32 +18,37 @@ import Hintman.Hint (getAllComments)
 
 
 data Prs = Prs
-    { pr1  :: !PrInfo
-    , pr2  :: !PrInfo
-    , pr3  :: !PrInfo
-    , pr24 :: !PrInfo
-    , pr30 :: !PrInfo
-    , pr36 :: !PrInfo
+    { pr2     :: !PrInfo
+    , pr3     :: !PrInfo
+    , pr4     :: !PrInfo
+    , pr24    :: !PrInfo
+    , pr30    :: !PrInfo
+    , pr36    :: !PrInfo
+    , forkPr1 :: !PrInfo
     }
 
-owner :: Owner
-owner = Owner "kowainik"
+kowainik :: Owner
+kowainik = Owner "kowainik"
 
 repo :: Repo
 repo = Repo "hintman-target"
 
 fetchPrs :: IO Prs
 fetchPrs = usingLoggerT richMessageAction $ do
-    pr1  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
-    pr2  <- makePr (PrNumber 3) (Sha "vrom911-patch-2")
-    pr3  <- makePr (PrNumber 4) (Sha "chshersh/parse-error")
+    pr2  <- makePr (PrNumber 2) (Sha "vrom911-patch-1")
+    pr3  <- makePr (PrNumber 3) (Sha "vrom911-patch-2")
+    pr4  <- makePr (PrNumber 4) (Sha "chshersh/parse-error")
     pr24 <- makePr (PrNumber 24) (Sha "chshersh/change-one-in-big")
     pr30 <- makePr (PrNumber 30) (Sha "chshersh/test-hintman")
     pr36 <- makePr (PrNumber 36) (Sha "chshersh-patch-1")
+    forkPr1 <- makePrFromFork (Owner "popzxc") (PrNumber 1) (Sha "no-newline-at-file-end")
```

Comment:
It's quite suprirpsing that this PR has number 1. Would it work if you change owner to Kowainik and number to 39?

* https://github.com/kowainik/hintman-target/pull/39

If no, is this documented somewhere or can be checked?

Reply:
I tried to do this at first (as you can see from https://github.com/kowainik/hintman-target/pull/39 existence :) ), but unfortunately the current implementation [attempts](https://github.com/kowainik/hintman/blob/master/src/Hintman/Hint.hs#L63) to retrieve the diff using the owner of PR as the base repo.

To get this working we need to add something like `prBaseRepo` field to the `PrInfo` type, and I believe this is a subject for a different PR.

If you agree that it should be done, I totally don't mind to create one another PR implementing that :)

Reply:
Created #158 for this.